assets/queries/cloudFormation/security_groups_meta_ip/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+CxPolicy [ result ] {
+   document := input.document
+   resources := input.document[i].Resources[name]
+   check_security_groups_ingress(resources.Properties)
+   
+      result := {
+                "documentId": 		input.document[i].id,
+                "searchKey":      sprintf("Resources.%s.Properties", [name]),
```

Comment:
The searchKey could be more specific pointing to Resources.%s.Properties.SecurityGroupIngress.CidrIp=%s", [name, ip]